docs/cron-backfill.md

```diff
@@ -10,7 +10,7 @@
 2. Create your cron workflow to run daily and invoke that template.
 3. Create a backfill workflow that uses `withSequence` to run the job for each date.
 
-This [full example](examples/cron-backfill.yaml) contains:
+This [full example](https://github.com/argoproj/argo-workflows/blob/master/examples/cron-backfill.yaml) contains:
```

Comment:
This is actually intended as you should view the docs from the website instead and the link should work there: https://argoproj.github.io/argo-workflows/cron-backfill/

Reply:
Oh I see! I saw the doc from github ( https://github.com/argoproj/argo-workflows/blob/master/docs/cron-backfill.md ). I'll close this pr.